test/e2e/index.test.ts

```diff
@@ -777,6 +777,76 @@ async function initializeEnvironment(
           // check that selecting a day doesn't change the day text
           expect(await day11.textContent()).to.equal('11');
         });
+
+        describe('Focus behavior', () => {
```

Comment:
I suspect a regular karma test would be enough here (so better since faster to run)? We are doing relatively simple events.

Reply:
If I'm not mistaken, I tried writing unit tests initially, but failed to make them work.
I didn't try skipping them in `jest` though. I'll look into that.

Reply:
I've tried karma tests and they have the same problem...
Maybe it's some sort of a race condition with timeouts, but on those tests, the `document.activeElement` is resolved/set on the button element. 🙈